ci-operator/step-registry/ibmcloud/provision/kms-key/ibmcloud-provision-kms-key-commands.sh

```diff
@@ -42,31 +65,18 @@ cat >${key_file} <<EOF
 }
 EOF
 
-instance_name="${CLUSTER_NAME}-kp"
-ibmcloud resource service-instance-create ${instance_name} kms tiered-pricing ${region}
+set -x
 
-id=$(ibmcloud resource service-instance ${instance_name} --id -q | awk '{print $2}')
-echo "$(jq --arg idarg "$id" '. += {"id": $idarg}' $key_file)" > $key_file
+createKey "${CLUSTER_NAME}-kp-master" ${region}
```

Comment:
This step should also have env setting like https://github.com/openshift/release/pull/47656/files#diff-fa66772b4e70848940b69b72a1658edd9d5aaad86fffc4b05d8e00b3abe80c38R8, only when `IBMCLOUD_DEFAULT_MACHINE_ENCRYPTION_KEY`, `IBMCLOUD_COMPUTE_ENCRYPTION_KEY` or `IBMCLOUD_CONTROL_PLANE_ENCRYPTION_KEY` is enabled, then provision the necessary keys.